### PR TITLE
[hi] Localize cloud-computing.md

### PR DESCRIPTION
**Pull request by kumarankit999**
Localize the Cloud Computing Part in the Hindi language

### Describe your changes
I have Localized the whole Cloud Computing file in the Hindi language

### Checklist before opening this PR (put `x` in the checkboxes)
- [x ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/hi/cloud-computing.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+हम आम तौर पर निजी और सार्वजनिक क्लाउड के बीच अंतर करते हैं, यह इस बात पर निर्भर करता है कि क्लाउड इंफ्रास्ट्रक्चर विशेष रूप से किसी संगठन के लिए समर्पित है या खुली सार्वजनिक सेवाओं के लिए साझा किया गया है।
+
+## यह समस्या का समाधान करता है
+कंप्यूटिंग शक्ति का विस्तार करने का प्रयास करते समय संगठनों को पारंपरिक रूप से दो मुख्य चुनौतियों का सामना करना पड़ता है।
+वे या तो अपने भौतिक सर्वर और नेटवर्क को होस्ट करने के लिए (नई) सुविधाओं का अधिग्रहण, समर्थन और डिजाइन कर सकते हैं या मौजूदा का विस्तार और रखरखाव कर सकते हैं।
+क्लाउड कंप्यूटिंग संगठनों को उनकी कुछ कंप्यूटिंग आवश्यकताओं को आउटसोर्स करने की अनुमति देकर उस चुनौती को हल करती है।
+
+## यह कैसे मदद करता है
+
+क्लाउड प्रदाता संगठनों को ऑन-डिमांड कंप्यूटिंग संसाधनों को किराए पर लेने और उपयोग के लिए भुगतान करने की अनुमति देते हैं, जिससे दो प्रमुख लाभ मिलते हैं।
```
Comment (aj11anuj) [inline content/hi/cloud-computing.md:18]:
```suggestion
क्लाउड के प्रदाता संगठनों को ऑन-डिमांड कंप्यूटिंग के संसाधनों को किराए पर लेने और उपयोग के लिए भुगतान करने की अनुमति देते हैं, जिससे दो प्रमुख लाभ मिलते हैं।
```

File: content/hi/cloud-computing.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+सबसे पहले, संगठन नए भौतिक बुनियादी ढांचे पर प्रतीक्षा, योजना और संसाधन खर्च किए बिना अपने उत्पाद या सेवा पर ध्यान केंद्रित कर सकते हैं। और दूसरा, वे आवश्यकतानुसार ऑन-डिमांड स्केल कर सकते हैं।
```
Comment (aj11anuj) [inline content/hi/cloud-computing.md:19]:
```suggestion
पहला, संगठन नए भौतिक इन्फ्रा़स्ट्रक्चर पर प्रतीक्षा, योजना और संसाधन के खर्च किए बिना अपने उत्पाद या सेवा पर ध्यान केंद्रित कर सकते हैं। और दूसरा, वे आवश्यकतानुसार ऑन-डिमांड स्केल कर सकते हैं।
```

File: content/hi/cloud-computing.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+क्लाउड कंप्यूटिंग संगठनों को आवश्यकतानुसार अधिक या कम बुनियादी ढाँचा अपनाने की अनुमति देता है।
```
Comment (aj11anuj) [inline content/hi/cloud-computing.md:20]:
```suggestion
क्लाउड कंप्यूटिंग संगठनों को आवश्यकतानुसार अधिक या कम इन्फ्रा़स्ट्रक्चर अपनाने की अनुमति देता है।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from me

**Review by kumarankit999 — COMMENTED** [review]
Now I will do it soon.

**Comment by aj11anuj** [comment]
@kumarankit999 gentle reminder to work on this PR

**Comment by kumarankit999** [comment]
Yes!

**Comment by aj11anuj** [comment]
@kumarankit999 gentle ping reminder

**Comment by bishal7679** [comment]
Since this PR has no activity for a long while so closing it!
Feel free to open whenever you'd get bandwidth to work :+1:

**Pull request closed without merging**